app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -6,12 +6,18 @@
     android:layout_height="wrap_content"
     android:orientation="vertical">
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:text="@string/synchronization_credentials_explanation" />
+
     <TextView
         android:id="@+id/createAccountWarning"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:text="@string/gpodnetauth_encryption_warning"
-        android:textColor="#F44336"
+        android:textColor="@color/download_failed_red"
```

Comment:
In #6237, I removed `@color/download_failed_red`. Instead, `?attr/icon_red` adapts to the theme colors.